### ISSUE TITLE
Request: Allow for movement of overlay

### ISSUE THREAD

**Issue by CptBroon** [Cities2Modding/MapImageLayer#2]
Fantastic mod so far, but I can't seem to be able to move my screenshot around in the map, which makes 'tiling' screenshots impossible - this would be a great addition, in my opinion - any chance of this being added?

**Comment by andylmfao** [comment]
+1! no real point of the downscaling if you can't move it, also, and since we can't export a full high res image of the map in any way, It's impossible to line things up with existing roads.

**Comment by jamesjiang221** [comment]
Agreed, the ability to move and rotate the image will save so much time to match the map in game, I have given up using Photoshop to adjust to match the map in game. Much appreciate it if this mod could be updated with these two functions.

**Comment by BrokVid** [comment]
Bro, please implement this fix in the mod, it's a very useful feature since I'm stitching the map image with a separate app, I definitely need to move it around
